util/topgen/merge.py

```diff
@@ -592,6 +592,10 @@ def is_unmanaged_clock(top: OrderedDict, clock: str):
     return clock in top['unmanaged_clocks']._asdict()
 
 
+def is_unmanaged_reset(top: OrderedDict, reset: str):
+    return reset in top['unmanaged_resets']._asdict()
```

Comment:
I'd probably defined `__contains__` for the `UnmanagedResets` class. For example:
```python
class Foo:
    def __init__(self):
        self.d = {1: 2}

    def __contains__(self, k: int) -> bool:
        return k in self.d


foo = Foo()
print(1 in foo)
print(2 in foo)
```

Reply:
In a follow up, I'll do that for the clocks as well.